sources/legal/latest/policies/acceptable-use-policy.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Apify Acceptable Use Policy
+description: Apify's acceptable use policy describes activities that are prohibited on the Apify platform and on our websites.
+sidebar_label: Acceptable Use Policy
+sidebar_position: 1
+category: legal
+slug: /acceptable-use-policy
+---
+
+<!-- vale off -->
+
+# Apify Acceptable Use Policy
+
+Apify Technologies s.r.o., with its registered seat at Vodičkova 704/36, Nové Město, 110 00 Prague 1, Czech Republic, Company ID No.: 04788290, registered in the Commercial Register kept by the Municipal Court of Prague, File No.: C 253224 (hereinafter referred to as “**we**” or “**Apify**”), is committed to making sure that the Platform and the Website are being used only for legitimate and legal purposes.
+
+By agreeing to our General Terms and Conditions or simply by using the Platform or the Website, you also agree to be bound by this Acceptable Use Policy.
+
+## A. General Provisions
```

Comment:
small nitpick , is there any reason why there are three style of headings?

With lettered lists, numbered lists and without lettering/numbering?